### DISCUSSION TITLE
How to use next-auth with mongodb v6.4?

### DISCUSSION THREAD

**Discussion by YuriGor** [nextauthjs/next-auth#10196] [Help]
When I install latest stable next-js + mongodb adapter - it complains about missing 5^ 4^ mongodb client version (I have v6.4)
In the NPM I see v1.1.3 is the latest published version of adapter.
But in GitHub project there is already v2.4 exists. 
Should I install next-auth v5.0.0-beta.13 + mongodb-adapter v2.4 to have 6.4 mongodb client?
I can install v5.0.0-beta.13 from NPM but how to install adapter?

**Comment by ryanwellsdev** [comment]
I have the same question...
